### PR TITLE
feature: support Business Partners module

### PR DESCRIPTION
**Pull request by samzong**
**What type of PR is this?**

/kind feature

Two types of display are supported, please ignore the content

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

@RainbowMango @wawa0210

### CODE REVIEW

**Comment by samzong** [comment]
https://deploy-preview-531--karmada.netlify.app

**Comment by samzong** [comment]
@Arhell @rgrupesh @RainbowMango @wawa0210 @windsonsea 

Which way would you prefer?

- A: Similar to Adporter
- B: Similar to Docusaurus 'Showcase'

**Comment by wawa0210** [comment]
> @Arhell @rgrupesh @RainbowMango @wawa0210 @windsonsea
> 
> Which way would you prefer?
> 
> * A: Similar to Adporter
> * B: Similar to Docusaurus 'Showcase'

Amazing, in terms of presentation, I tend to choose plan B

File: src/pages/business-adopters/index.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+import React from "react";
+import Translate, { translate } from "@docusaurus/Translate";
+import { Typography } from "@douyinfe/semi-ui";
+
+export default function Partners() {
+  const { Text } = Typography;
+
+  const PartnersList = [
+    {
+      image: "/assets/images/huawei-501bd8d1790fd7f58a3a5af6c1b9107a.png",
```
Comment (RainbowMango) [inline src/pages/business-adopters/index.tsx:27]: Does this image reference a picture from the repo?

File: src/pages/business-adopters/index.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
...Isg/g9GBe5hYKV01MBmyviQDhuxcS0uxbpz8V8xH8H/s8QYXCfNsB+ijZtCvJcj8e+D2guIFes0saSPbX5J2qaQtpbUtOJiYniGhKBJ4v1g59JkM9DKr5eiPZ9LcBPQzs8jnEg/VvQ8LByAr7PqK+O8VkQT86O1tZWnzjI3DNo0KAlcElOPPjzj8R9JjA9nZQMDXwDitUPDbkdiFf6IooFqE04HfctcE4jPNA4HbyUgw6t9MpYOSXhDKA8ugCrQb1/6Ex1/gK2BOXWCbBcxHeCxofI+w55mdjLOg+hkw7M/ELkPybgG0bR4W+Bh6dQVnVFF4LWDtB+G7T7YDvgX1HYrUCIb8U6kntpoLKysgFyOAK4c9sDdPqi/i0o/xZgZ+K+BTB/bubvwDsPt/MC/hugSRONVsT/BXe2KyvowJfp9yAGhn2NKzxwKQHhNGMjcob2LAwY/18wfI1XAR8A2j6gFWEVBbxcj/gbatpXCEWcCYUVlUO1ZK+CD48n3YgO2tICHkagM4/Amg2A26gRcD/DCq5zw1OFwUreCyV2bw6qcG0IuifB3wRYPQ83Cxk9Dt4e1OIbpUHnIbiiJ9R8WNl+UNpq0AjYiqANAelCyC5JZU4vRAM2QBBjtcpCuHjaZyPYqleOYCj7MW7aizG80LgNwJmD2+37dZAbkX+3VlkID3AFliAP0YdxawfRhCJetCL6NPB91SXis5kzZz4HANE2NMHggdzGtVplIUJQzuXI1z1HDDiNwTxcLpURL9exjp8C1iTC1ThoHMLtU8YqrjYMSKuoEJ5qn08MGGgB2iHcH0DbtX6cSLgvaD7HUzQLZcj95LgydiL8Izr4T9hgvABP8Sy1AJa2f653XsX1tN8JvGzQovMw+FIq34V0FYT+AlkElYZRSOdjMJNZiLKXAmcw7iTE60GHnvpNWAh7sby83D2IpN1lnKd5RqUHvGq0d7GaFkO0YyjacTvojQXeGIS0SUr8fY4jny/QMr2IL8bpGClWwRehzNkoQ4PkGoT0vwlFcGEtkN9KwJwPPCz5BijeX8XyFId8hsHS3Q0atLE7FCBy5W/jLgIdspSLEHdeeAhJjtZlScCSgCUBSwKWBCwJWBKwJGBJwJKAJQFLApYELAlYErAkYEnAkoAlAUsClgQsCVgSsCRgSSBkCfwTBEXYlhQii0kAAAAASUVORK5CYII=",
```
Comment (RainbowMango) [inline src/pages/business-adopters/index.tsx:38]: Can we refer to a picture from the repo or a URL?

**Review by RainbowMango — COMMENTED** [review]
Excellent!
Just asking some questions as I'm thinking if it is easy to add more items.

**Comment by windsonsea** [comment]
[Plan B](https://deploy-preview-531--karmada.netlify.app/business-adopters) +1

File: src/pages/business-adopters/index.tsx (hunk 1 above)
Comment (samzong) [reply src/pages/business-adopters/index.tsx:27]: pic will be saved into this repository.

File: src/pages/business-adopters/index.tsx (hunk 2 above)
Comment (samzong) [reply src/pages/business-adopters/index.tsx:38]: Yes, it's just a sample of different picture styles.

**Comment by samzong** [comment]
> Excellent! Just asking some questions as I'm thinking if it is easy to add more items.

Good question! I will create a simple file of the partners list, like the current supporter list.

**Comment by RainbowMango** [comment]
If it possible to share the images by supporter and parterner?  I think we don't need to maintain multiple copies of the same logo.

**Comment by samzong** [comment]
> If it possible to share the images by supporter and parterner? I think we don't need to maintain multiple copies of the same logo.

Okay, We can merge all the icons, whether it is a supporter or partner. used freely on different pages.

**Comment by samzong** [comment]
The partner page has been ready,
Next, I need information about the current partners.

@RainbowMango @wawa0210

**Comment by RainbowMango** [comment]
@samzong Can you help put [Union Big Data](https://github.com/karmada-io/community/issues/64) onto the list?

**Comment by samzong** [comment]
> @samzong Can you help put [Union Big Data](https://github.com/karmada-io/community/issues/64) onto the list?

@RainbowMango okay, I will uprade in this PR.

File: src/pages/partners.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                <Translate id="partners.title">
+                  Become a Karmada Partner
+                </Translate>
+              </h1>
+            </div>
+            <div className="col col--12">
+              <p className={styles.font}>
+                <Translate id="partners.description1">
+                  Welcome to join us and look forward to your arrival. For more
+                  information, Create an issue in the Karmada repository.
```
Comment (RainbowMango) [inline src/pages/partners.tsx:30]:
```suggestion
                  Vetted partners with deep experience helping enterprises successfully adopt Karmada.
                  Interested in becoming a partner?
```
Referred to the words in https://kubernetes.io/partners/. 
Can we add a link to partners' guidelines? The link should be https://github.com/karmada-io/community/blob/main/commercial-partner.md
Even though the

File: src/data/partnersData.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+import huawei from "../../static/img/supporters/huawei.png";
+import daocloud from "../../static/img/supporters/DaoCloud.png";
+import DSK from "../../static/img/supporters/DSK.png";
+
+const PartnersList = [
+  {
+    image: huawei,
+    title: "Huawei",
+    webURL: "https://www.huaweicloud.com/intl/en-us/",
```
Comment (RainbowMango) [inline src/data/partnersData.tsx:12]:
```suggestion
    webURL: "https://www.huaweicloud.com/intl/en-us/product/ucs.html",
```
I guess a service URL might be more helpful to users.

File: src/data/partnersData.tsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    description: (
+      <Translate id="partners.huawei.description">
+        Huawei Cloud is a leading cloud service provider in China and Asia
+        Pacific.
```
Comment (RainbowMango) [inline src/data/partnersData.tsx:16]:
```suggestion
        UCS provides consistent experience in cloud native application deployment, management, and ecosystem. Cloud native applications can freely run across regions and clouds with intelligent traffic distribution.
```
Get the introduction from https://www.huaweicloud.com/intl/en-us/product/ucs.html.

File: i18n/zh/code.json [hunk 6]
```diff
@@ -193,5 +193,14 @@
   "theme.SearchBar.label": {
     "message": "查询",
     "description": "The ARIA label and placeholder for search button"
+  },
+  "partners.huawei.description": {
+    "message": "构建无处不在的云原生服务，通过智能流量分配跨区域和跨云运行应用程序。"
```
Comment (RainbowMango) [inline i18n/zh/code.json:198]:
```suggestion
    "message": "Ubiquitous Cloud Native Service（UCS）是业界首个分布式云原生产品，为企业构建云原生业务部署、管理、应用生态的全域一致性体验 ，实现客户在使用云原生应用时，感受不到地域、跨云、流量的限制，让云原生的能力进入企业的每一个业务场景，加速千行百业拥抱云原生。"
```
Refer to https://www.huaweicloud.com/intl/zh-cn/product/ucs.html.

File: i18n/zh/code.json [hunk 7]
```diff
@@ -193,5 +193,14 @@
...
+  },
+  "partners.daocloud.description": {
+    "message": "使用 Karmada 构建更具弹性的混合多云平台。"
+  },
+  "partners.DSK.description": {
+    "message": "使用 Karmada 构建多策略和多地域的智能分布式容器平台。"
```
Comment (RainbowMango) [inline i18n/zh/code.json:204]: @pandaoknight
Please check and confirm.

File: src/data/partnersData.tsx [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+      </Translate>
+    ),
+  },
+  {
+    image: DSK,
+    title: "DSK",
+    webURL: "https://www.unionbigdata.com/",
+    description: (
+      <Translate id="partners.DSK.description">
+        Use Karmada to build a intelligent and distributed container platform with multi-policy and multi-region.
```
Comment (RainbowMango) [inline src/data/partnersData.tsx:36]: @pandaoknight Please confirm it.

File: src/pages/partners.tsx (hunk 3 above)
Comment (samzong) [reply src/pages/partners.tsx:30]: wait for karmada-io/community#65 Merged.

Comment (RainbowMango) [reply src/pages/partners.tsx]: Merged. See https://github.com/karmada-io/community/blob/main/community-partner.md.

File: src/data/partnersData.tsx (hunk 5 above)
Comment (RainbowMango) [reply src/data/partnersData.tsx]: Seems the description can not fully show out at the https://deploy-preview-531--karmada.netlify.app/partners/.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

Great thanks to @samzong.
Let's merger the first version.

File: src/data/partnersData.tsx (hunk 8 above)
Comment (pandaoknight) [reply src/data/partnersData.tsx:35]: Sorry for the delayed response.
The descriptions in both languages are clear. 

Please change the company name from **DSK** to **Unionbigdata**; this has been confirmed with our business team.

Comment (RainbowMango) [reply src/data/partnersData.tsx:35]: Got it. Working on it. See https://github.com/karmada-io/website/pull/549.

**Pull request merged**